packages/vaadin-grid/test/accessibility.test.js

```diff
@@ -184,6 +183,59 @@ describe('accessibility', () => {
     });
   });
 
+  describe('treegrid', () => {
+    function hierarchicalDataProvider({ parentItem }, callback) {
+      // Let's use a count lower than pageSize so we can ignore page + pageSize for now
+      const itemsOnEachLevel = 5;
+
+      const items = [...Array(itemsOnEachLevel)].map((_, i) => {
+        return {
+          name: `${parentItem ? parentItem.name + '-' : ''}${i}`,
+          // Let's only have child items on every second item
+          children: i % 2 === 0
+        };
+      });
+
+      callback(items, itemsOnEachLevel);
+    }
+
+    beforeEach(() => {
+      grid = fixtureSync(`
+      <vaadin-grid item-id-path="name">
+        <vaadin-grid-tree-column path="name" header="" width="100px" flex-shrink="0"></vaadin-grid-tree-column>
+        <vaadin-grid-column path="name" width="200px" flex-shrink="0"></vaadin-grid-column>
+      </vaadin-grid>
```

Comment:
nit: indentation seems a bit wrong. 

```suggestion
        <vaadin-grid item-id-path="name">
          <vaadin-grid-tree-column path="name" header="" width="100px" flex-shrink="0"></vaadin-grid-tree-column>
          <vaadin-grid-column path="name" width="200px" flex-shrink="0"></vaadin-grid-column>
        </vaadin-grid>
```